radixdlt-java/src/main/java/com/radixdlt/client/core/network/RadixJsonRpcClient.java

```diff
@@ -99,7 +98,11 @@ public String getLocation() {
 		return wsClient.getEndpoint().url().toString();
 	}
 
-	public Observable<RadixClientStatus> getStatus() {
+	public Observable<RadixClientStatus> status() {
+		return wsClient.status();
+	}
+
+	public Optional<RadixClientStatus> getStatus() {
 		return wsClient.getStatus();
```

Comment:
Code smell here in that we need both synchronous and asynchronous versions. Something must be wrong somewhere...If this is being used for the pure NetworkState function this `status` should be `flatMapped` somewhere...